Status.sh

```diff
@@ -85,10 +85,32 @@ then
 branch="\Zb\Z1$(< /srv/brnch)\Zn"
 fi
 
+. /etc/free-dns.sh
+if [ "$HOSTNAME" = "" ]
+then
+FD="No hostname"
+else
+registered=$(nslookup $HOSTNAME|tail -n2|grep A|sed s/[^0-9.]//g)
+current=$(wget -q -O - http://checkip.dyndns.org|sed s/[^0-9.]//g)
+if [ ! "$registered" = "$current" ]
+then
+FD="Mismatch"
+else
+FD="Match"
+fi
+fi
+
+grep API_SECRET /etc/nsconfig | awk '{print $2}' > /tmp/apisecret
+FLine=$(</tmp/apisecret)
+IFS='"'
+read -a split <<< $FLine
+apisec=${split[1]}
```

Comment:
The code here is trying to use greps /split in order to get the apisecret.
This code will not work in many cases. The most obvious of them is if the code would say something like:
export api_sec='secret' 
That is single quote and not double quote.
The web server PR will change that to single quote, as this is what most people want.

The solution to this is very simple, replace the code getting apisec with this.
. /etc/nsconfig
and then use it (for example):
echo $ API_SECRET

Reply:
Thanks for the tip.
Done